test/StdCheats.t.sol

```diff
@@ -247,6 +247,14 @@ contract StdCheatsTest is Test {
             vm.createSelectFork(rpcUrl);
         }
     }
+
+    function testAssumeNoPrecompilesL1(address addr) external {
+        assumeNoPrecompiles(addr, stdChains["optimism_goerli"].chainId);
+        assertTrue(
+            addr < address(1) || (addr > address(9) && addr < address(0x4200000000000000000000000000000000000000))
+                || addr > address(0x4200000000000000000000000000000000000800)
+        );
+    }
```

Comment:
Why are `address(1)`-`address(9)` here?

```suggestion
    function testAssumeNoPrecompiles(address addr) external {
        assumeNoPrecompiles(addr, stdChains["optimism_goerli"].chainId);
        assertTrue(
           addr < address(0x4200000000000000000000000000000000000000))
                || addr > address(0x4200000000000000000000000000000000000800)
        );
    }
```

Reply:
AFAIK all EVM compatible chains will need to have the same precompiles at those addresses, otherwise they wouldn't be EVM compatible. Test name fixed in f42f04d

Reply:
Makes sense! Thanks for the info.